manifests/t/timedomain/acestudio/1.0.3/timedomain.acestudio.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: timedomain.acestudio
+PackageVersion: 1.0.3
+PackageLocale: en-US
+Publisher: timedomAin, Inc.

```

```suggestion
Publisher: timedomAin
```
This must match Control Panel.